sdcm/cluster.py

```diff
@@ -2186,10 +2186,16 @@ def start_scylla(self, verify_up=True, verify_down=False, timeout=300):
     def stop_scylla_server(self, verify_up=False, verify_down=True, timeout=300, ignore_status=False):
         if verify_up:
             self.wait_db_up(timeout=timeout)
-        if not self.is_ubuntu14():
```

Comment:
I think we can remove this code on the way, we don't support ubuntu14 anymore.

Reply:
ok